src/dolphin/util.ts

```diff
@@ -65,3 +67,29 @@ export async function findUserFolder(type: DolphinLaunchType): Promise<string> {
 
   return userPath;
 }
+
+export async function addGamePathToInis(gameDir: string): Promise<void> {
+  await addGamePathToIni(DolphinLaunchType.NETPLAY, gameDir);
+  await addGamePathToIni(DolphinLaunchType.PLAYBACK, gameDir);
+}
+
+async function addGamePathToIni(type: DolphinLaunchType, gameDir: string): Promise<void> {
+  const userFolder = await findUserFolder(type);
+  const iniPath = path.join(userFolder, "Config", "Dolphin.ini");
+  if (fs.pathExistsSync(iniPath)) {
```

Comment:
Since this is an async function, we should avoid the usage of the `Sync` api calls.

```suggestion
  if (await fs.pathExists(iniPath)) {
```